### PR TITLE
[bitnami/etcd] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/etcd chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/etcd/values.yaml [hunk 1]
```diff
@@ -462,24 +467,24 @@ service:
...
   ##
   clusterIP: ""
-  ## @param service.port etcd client port
+  ## @param service.ports.client etcd client port
+  ## @param service.ports.peer etcd peer port
+  ##
+  ports:
+    client: 2379
+    peer: 2380
+  ## @param service.nodePorts [object] Specify the nodePort(s) value(s) for the LoadBalancer and NodePort service types.
```
Comment (carrodher) [inline bitnami/etcd/values.yaml:476]:
```suggestion
  ## @param service.nodePorts.client Specify the nodePort client value for the LoadBalancer and NodePort service types.
  ## @param service.nodePorts.peer Specify the nodePort peer value for the LoadBalancer and NodePort service types.
```

File: bitnami/etcd/values.yaml [hunk 2]
```diff
@@ -495,9 +500,26 @@ service:
...
+  ## @param service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+  ## If "ClientIP", consecutive client requests will be directed to the same Pod
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+  ##
+  sessionAffinity: None
+  ## @param service.sessionAffinityConfig Additional settings for the sessionAffinity
+  ## sessionAffinityConfig:
+  ##   clientIP:
+  ##     timeoutSeconds: 300
+  sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/etcd/values.yaml:522]:
```suggestion
  ##     timeoutSeconds: 300
  ##
  sessionAffinityConfig: {}
```

File: bitnami/etcd/values.yaml [hunk 3]
```diff
@@ -462,24 +467,24 @@ service:
   ## clusterIP: None
   ##
   clusterIP: ""
-  ## @param service.port etcd client port
+  ## @param service.ports.client etcd client port
+  ## @param service.ports.peer etcd peer port
+  ##
+  ports:
+    client: 2379
+    peer: 2380
```
Comment (juan131) [inline bitnami/etcd/values.yaml:475]: These changes are affecting other parts that you didn't take into account (apart from `svc.yaml`):
```console
$ ag "service.port" bitnami/etcd/templates
bitnami/etcd/templates/NOTES.txt
42:etcd can be accessed via port {{ .Values.service.port }} on the following DNS name from within your cluster:
48:    kubectl run {{ template "common.names.fullname" . }}-client --restart='Never' --image {{ template "etcd.image" . }}{{- if .Values.auth.rbac.enabled }} --env ROOT_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "common.names.fullname" . }} -o jsonpath="{.data.etcd-root-password}" | base64 --decode){{- end }} --env ETCDCTL_ENDPOINTS="{{ template "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.service.port }}" --namespace {{ .Release.Namespace }} --command -- sleep infinity
71:    echo "etcd URL: http://$SERVICE_IP:{{ .Values.service.port }}/"
75:    kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "common.names.fullname" . }} {{ .Values.service.port }}:{{ .Values.service.port }} &
76:    echo "etcd URL: http://127.0.0.1:{{ .Values.service.port }}"
99:    export ETCDCTL_ENDPOINTS=https://{{ template "common.names.fullname" . }}-0:{{ .Values.service.port }}

bitnami/etcd/templates/statefulset.yaml
161:              value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME).{{ $etcdHeadlessServiceName }}.{{ .Release.Namespace }}.svc.{{ $clusterDomain }}:{{ .Values.containerPorts.client }},{{ $etcdClientProtocol }}://{{ $etcdFullname }}.{{ .Release.Namespace }}.svc.{{ $clusterDomain }}:{{ .Values.service.port }}"
$ ag "service.peerPort" bitnami/etcd/templates/
bitnami/etcd/templates/svc.yaml
41:    - name: {{ default "peer" .Values.service.peerPortNameOverride | quote }}
42:      port: {{ .Values.service.peerPort  }}

bitnami/etcd/templates/svc-headless.yaml
31:    {{- if .Values.service.peerPortNameOverride }}
33:    - name: {{ .Values.service.peerPortNameOverride }}-ssl
35:    - name: {{ .Values.service.peerPortNameOverride }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]

File: bitnami/etcd/values.yaml (hunk 3 above)
Comment (migruiz4) [reply bitnami/etcd/values.yaml:475]: Thanks you Juan, I forgot to fix the references to `service.port`

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**